### ISSUE TITLE
Touch-events select the "text" -> unwanted behavior

### ISSUE THREAD

**Issue by SVAR7BERG** [jamesmfriedman/rmwc#160]
## RMWC or MDC?

To be honest, I don't really know. But I've tried other MD implementations for react which don't show this behavior.

### What RMWC Version are you using?

> "rmwc": "^1.3.0"

### What browser(s) and React Version is this bug affecting?

> Chrome on Android. Couldn't test on others yet.

### What build system are you using?

> Webpack

### What are the steps to reproduce the bug?

> Clicking for example an icon, select the "icon text" which leads to android offering to search for that string. You can see what I mean in this video I recorded: https://youtu.be/_MjQHMgR8GE

### What is the expected behavior?

> I guess just that it shouldn't select the text

### What is the actual behavior?

> See the video, I posted above

**Comment by jamesmfriedman** [comment]
This is very strange behavior! RMWC only renders what MDC does. Since this is definitely DOM related, I would say this is likely an issue with the library. One suggestion I would have is to just globally disable selection events for icons in css.

**Comment by jamesmfriedman** [comment]
@SVAR7BERG did that response help you at all?

**Comment by maccuaa** [comment]
I ran into this recently as well, specifically with the Tab component when using icons.  It's because of Chrome's [Touch to Search](https://developers.google.com/web/updates/2015/10/tap-to-search) feature.

When comparing the DOM from RMWC to the MDC examples, the only difference is that they use an anchor tag instead of a div. Because the icon is just text in a div element, Chrome doesn't see it as an icon element so the search feature pops up.

For me, I just used the tag prop and set the Tab element to be an anchor tag `tag="a"`  and added an href.  In the link above they also list other ways to prevent the Touch To Search from popping up.
